### PR TITLE
Fixing UI test_errata

### PR DESCRIPTION
**Pull request by akhil-jha**
1. Adding coverage for BZ2029192
2. Minor Assertion updates.
3. Removing 1 test as BZ is on WONTFIX

### CODE REVIEW

**Comment by akhil-jha** [comment]
trigger: robottelo
pytest: tests/foreman/ui/test_errata.py

**Review by jyejare — CHANGES_REQUESTED** [review]
ACK pending changes requested !

File: tests/foreman/ui/test_errata.py [hunk 1]
```diff
@@ -228,6 +228,8 @@ def test_end_to_end(session, module_org, module_repos_col, vm, default_sat):
 
     :parametrized: yes
 
+    :BZ: 2029192
```
[the remark this answers is not in the document]
Comment (akhil-jha) [reply tests/foreman/ui/test_errata.py:232]: yes

File: tests/foreman/ui/test_errata.py [hunk 2]
```diff
@@ -251,6 +253,7 @@ def test_end_to_end(session, module_org, module_repos_col, vm, default_sat):
     assert _install_client_package(vm, FAKE_1_CUSTOM_PACKAGE)
     value = 'has type = security'
     with session:
+        session.settings.update('name = remote_execution_by_default', 'Yes')  # BZ 2029192
```
[the remark this answers is not in the document]
Comment (akhil-jha) [reply tests/foreman/ui/test_errata.py]: Changes made. Could you please re-review it?

File: tests/foreman/ui/test_errata.py [hunk 3]
```diff
@@ -720,53 +725,6 @@ def test_positive_content_host_search_type(session, erratatype_vm):
         assert errata_ids == sorted(FAKE_11_YUM_ENHANCEMENT_ERRATUM)
 
 
-@pytest.mark.skip_if_open("BZ:1655130")
-@pytest.mark.tier3
-def test_positive_content_host_errata_details(session, erratatype_vm, module_org, test_name):
```
[the remark this answers is not in the document]
Comment (akhil-jha) [reply tests/foreman/ui/test_errata.py:725]: The test is actually failing.

File: tests/foreman/ui/test_errata.py [hunk 4]
```diff
@@ -852,7 +810,6 @@ def test_positive_show_count_on_content_host_details_page(session, module_org, e
 
 @pytest.mark.tier3
 @pytest.mark.upgrade
-@pytest.mark.skip_if_open('BZ:2013093')
```
[the remark this answers is not in the document]
Comment (akhil-jha) [reply tests/foreman/ui/test_errata.py:855]: Done

File: tests/foreman/ui/test_errata.py [hunk 5]
```diff
@@ -932,14 +889,17 @@ def test_positive_filtered_errata_status_installable_param(
             }
             for key in actual_values:
                 assert expected_values[key] in actual_values[key], 'Expected text not found'
-            _set_setting_value(errata_status_installable, False)
+            _set_setting_value(errata_status_installable, True)
```
[the remark this answers is not in the document]
Comment (akhil-jha) [reply tests/foreman/ui/test_errata.py]: Same as above

**Review by jyejare — APPROVED** [review]

**Review by latran — APPROVED** [review]
ACK

File: tests/foreman/ui/test_errata.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (pondrejk) [reply tests/foreman/ui/test_errata.py:855]: sorry but I don't see it added, did you push?

**Review by pondrejk — APPROVED** [review]
ack pending docstring fix

File: tests/foreman/ui/test_errata.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (akhil-jha) [reply tests/foreman/ui/test_errata.py:855]: Oops, I added in the wrong place!! :open_mouth: 
Done now.

**Pull request merged**